docs/v-client-library-docs.rst

```diff
@@ -23,67 +23,6 @@ fn catalog_name_from_path
 
 
 
-fn new_varchar_value
```

Comment:
I think the build is failing because there are coming are not being written in a deterministic order. I think you need to add a sort here: https://github.com/elliotchance/vsql/blob/main/scripts/generate-v-client-library-docs.vsh#L32

Reply:
All right, they are now sorted with `dcs_contents.sort_by_kind()` .

Reply:
For me, running `make clean-docs docs` does not change the order.

Reply:
Looks like there are some legitimate build failures now.

Reply:
> Looks like there are some legitimate build failures now.

Indeed. Given that vsql has not changed since May, I think there is a codegen regression in V. I'll bisect later today to see when it happened and what caused it.